mkdocs.yml

```diff
@@ -28,9 +28,12 @@ nav:
 
   - Variational quantum algorithms:
     - qml/index.md
-    - Quantum circuit learning: qml/qcl.md
-    - Solving MaxCut with QAOA: qml/qaoa.md
-    - Tools for quantum machine learning: qml/qml_tools.md
+    - Tools for quantum machine learning:
```

Comment:
I would flatten this nested hierarchy of pages. Just feels easier to flow the reading from IMHO.